assignment.js

```diff
@@ -29,8 +29,21 @@ Use the following test cases to confirm your program meets the success criteria
     24
 ************/
 console.log('Problem 1:')
-
+console.log('')
 // Add your code below this line
+ let baseNumber = 2
+
+
+ // loop to run 12 multiples of the baseNumber
+ for (let i = 1; i < 13; i++){
+  let num = baseNumber * i
+  console.log(num)
+  }
+
+
```

Comment:
@jct117 Good Job implementing a for loop to solve this problem. I would recommend starting setting the variable `let i = 0` just so when the program runs the first number that is console logged is 0
take a look at this alternate approach to the solution [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L34)